compose.js

```diff
@@ -89,7 +89,11 @@ function mergeComposable(dstDescriptor, srcComposable) {
   combineProperty('deepConfiguration', merge);
   if (Array.isArray(srcDescriptor.initializers)) {
     if (!Array.isArray(dstDescriptor.initializers)) dstDescriptor.initializers = [];
-    dstDescriptor.initializers.push.apply(dstDescriptor.initializers, srcDescriptor.initializers.filter(isFunction));
+    srcDescriptor.initializers.forEach(i => {
```

Comment:
Would be using `reduce` better? Something like this without side effects and in my opinion somewhat easier to reason about.

```
dstDescriptor.initializers = srcDescriptor.initializers.reduce((result, init) => {
     if (isFunction(init) && !result.includes(init)) {
         result.push(init);
     }
     return result;
}, Array.isArray(dstDescriptor.initializers) ? dstDescriptor.initializers : []);
```